Prolangle/Languages/Lua.cs

```diff
@@ -0,0 +1,22 @@
+using Prolangle.Languages.Framework;
+
+namespace Prolangle.Languages;
+
+public class Lua : BaseLanguage
+{
+	public override Guid Id { get; } = Guid.NewGuid();
+	public override string Name => "Lua";
+	public override TypeSystem Typing => TypeSystem.Weak | TypeSystem.Dynamic | TypeSystem.Duck;
+	public override bool Compiled => false;
```

Comment:
If we do do #77, maybe this one should be JIT.

Reply:
So, postpone this PR until #77 is fixed?

Reply:
Nah… we can merge this one regardless.